cpp/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ option(TOPPRA_DEBUG_ON "Set logging mode to on." OFF)
 option(TOPPRA_WARN_ON "Enable warning messages." ON)
 option(OPT_MSGPACK "Serialization using msgpack " OFF)
 # Bindings
-option(PYTHON_BINDINGS "Build bindings for Python" ON)
+option(PYTHON_BINDINGS "Build bindings for Python" OFF)
```

Comment:
Please don't change the default flag needlessly. You can always use `-DPython_BINDINGS=OFF` during compilation.

Reply:
OK.